src/adapters/store.js

```diff
@@ -3,111 +3,169 @@ import Error from '../adapters/error'
 import {version} from '../../config/constants.yml'
 import ExtendedString from "../libs/string";
 
-let moduleConfig = nconf.get().store
+export default class Store {
 
-const AbStore = require(`../libs/${moduleConfig.name}`)
-const abstractStore = new AbStore(moduleConfig.endpoint)
+  constructor() {
+    // get store from window if already initialized
+    if (window.__store) {
+      return window.__store
+    }
+    // if store not initialized, use this
+    window.__store = this
 
-function Store() {
-  this.isRegisterd = false
-  if(!window.__existingStore) {
-    window.__existingStore = true
-    listen('store_poi', (poi) => {
-      this.add(poi)
-    })
-    listen('del_poi', (poi) => {
-      this.del(poi)
-    })
-    listen('store_center', (loc) => {
-      this.setLastLocation(loc)
-    })
-    listen('store_clear', () => {
-      this.clear()
-    })
+    // init stores
+    const LocalStore = require(`../libs/local_store`)
+    this.localStore = new LocalStore()
+    this.loggedIn = false
+    this.abstractStore = this.localStore
+    const MasqStore = require(`../libs/masq`)
+    this.masqConfig = nconf.get().masq
+    this.masqStore = new MasqStore(this.masqConfig)
+    const masqInitPromise = this.masqStore.onConnect()
+    const masqInitialized = false
```

Comment:
unused

Reply:
It's an error on my part, it was supposed to be an attribute used in checkInit